src/main/java/CycleTimeGUI.java

```diff
@@ -29,45 +40,89 @@ public CycleTimeGUI(int projectID,String authToken, String TAIGA_API_ENDPOINT,St
         this.firstDate = firstDate;
         this.lastDate = lastDate;
         System.out.println(projectID+"\n"+authToken+"\n"+TAIGA_API_ENDPOINT+"\n"+firstDate+"\n"+lastDate);
-//        return;
+        this.xAxis = new CategoryAxis();
+        this.yAxis = new NumberAxis(0, 12,1);
+        this.scatterChart = new ScatterChart<>(xAxis, new NumberAxis());
     }
 
 
 
     public static void main(String[] args) {
```

Comment:
not required

Reply:
This is required because if I initialize them in a method it will throw null value exception in the other methods

Reply:
can you share the error pls